test/all-theme-imports.html

```diff
@@ -10,6 +10,12 @@
 <link rel="import" href="../vaadin-date-picker.html">
 <link rel="import" href="../vaadin-dialog.html">
 <link rel="import" href="../vaadin-dropdown-menu.html">
+<link rel="import" href="../vaadin-grid-column.html">
```

Comment:
No import for `vaadin-grid.html`?

Reply:
It’s there below. On my system, `ls` does sort the files this way. I can’t notice any errors with that order, so I’d keep it as it is.